<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewMetadataDialog</class>
 <widget class="QDialog" name="ViewMetadataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Metadata</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblDescriptor">
     <property name="text">
      <string>List of descriptors :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lstDescriptors">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbImage"/>
   </item>
   <item>
    <widget class="QTextEdit" name="txtMetadata"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
